app/models/topic.rb

```diff
@@ -1287,6 +1267,43 @@ def self.ensure_consistency!
     Topic.where("pinned_until < now()").update_all(pinned_at: nil, pinned_globally: false, pinned_until: nil)
   end
 
+  def inherit_auto_close_from_category(timer_type: :close)
```

Comment:
I moved this whole method so it is close to the other topic timer methods in the code